### ISSUE TITLE
Getting undefined method `[]' for nil:NilClass while running

### ISSUE THREAD

**Issue by fireundubh** [JeffreyATW/mbfc_crawler#2]
```
Could not crawl source: http://mediabiasfactcheck.com/addicting-info/
undefined method `[]' for nil:NilClass
```

This repeats for all sources.

Also, the `source_domains` array doesn't appear to ever be used.

**Comment by JeffreyATW** [comment]
Looks like MBFC changed their pages. I'll take a look into it. Thanks!

**Comment by JeffreyATW** [comment]
Fixed in 35bf1a4e3b7e6d721673a34edced78417a47f063.
